### PR TITLE
[vrops-exporter] New NSX-T alerts

### PR DESCRIPTION
**Pull request by christopherhans**
Added alerts for NSX-T monitoring. Used 80/95% for warning/critical.

Waiting for Operations to check thresholds.

### CODE REVIEW

**Review by viennaa — APPROVED** [review]
lgtm, can't judge thresholds though

**Comment by christopherhans** [comment]
Changed alerts. Alert will be fired, if the usage count exceeds the maximum supported count.
Disabled absent alerts, as long as the nsxt adapter is not configured everywhere.

File: prometheus-exporters/vrops-exporter/alerts/nsxt.alerts [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+  rules:
+  - alert: NSXTDistributedFirewallSectionUsageLimitExceeded
+    expr: |
+        vrops_nsxt_sys_capacity_distributed_firewall_section_usage_count >
+        vrops_nsxt_sys_capacity_distributed_firewall_section_max_supported_count
+    for: 10m
+    labels:
+      severity: info
+      tier: vmware
+      service: compute
```
Comment (richardtief) [inline prometheus-exporters/vrops-exporter/alerts/nsxt.alerts:12]: Is this really `compute`? Or is there a `network` category?

Comment (richardtief) [reply prometheus-exporters/vrops-exporter/alerts/nsxt.alerts:12]: Same question applies for the following alerts.

**Review by richardtief — APPROVED** [review]
Thanks. Just one remark.

File: prometheus-exporters/vrops-exporter/alerts/nsxt.alerts (hunk 1 above)
Comment (christopherhans) [reply prometheus-exporters/vrops-exporter/alerts/nsxt.alerts:12]: yes, it is an InfraOps topic, therefore we need it in the #alert-vmware-warning channel.

Comment (christopherhans) [reply prometheus-exporters/vrops-exporter/alerts/nsxt.alerts:12]: (info first...)

Comment (richardtief) [reply prometheus-exporters/vrops-exporter/alerts/nsxt.alerts:12]: Alright.

Comment (christopherhans) [reply prometheus-exporters/vrops-exporter/alerts/nsxt.alerts:12]: wait... routing is based on the tier. my bad.

**Pull request merged**